keyboards/sawnsprojects/krush/krush65/hotswap/readme.md

```diff
@@ -0,0 +1,15 @@
+# Krush65 Hotswap
+
+![Krush65](https://imgur.com/56ZvWUmh.jpg)
+
+A 65% keyboard PCB made and sold by SawnsProjects with Nuxros in [mechkey.store](https://mechkey.store/blogs/store-updates/introducing-krush-65). 
+
+* Keyboard Maintainer: [Mai The San](https://github.com/MaiTheSan)
+* Hardware Supported: Krush65
+* Hardware Availability: [Facebook](https://mechkey.store/products/krush-65-hotswap-pcb)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```